controllers/index.js

```diff
@@ -0,0 +1,5 @@
+const { getAllTopics } = require('./topics.controller');
+const { getEndPoints } = require("./endPoints.controller");
+
+exports.getAllTopics = getAllTopics;
```

Comment:
Since we're never requiring all of the controllers into a file at once, there is no need to export all of these through an index file. I would recommend just exporting them from the file that they live in, otherwise you're adding a needless layer of complexity.